package-lock.json

```diff
@@ -6,7 +6,7 @@
     "": {
       "license": "EUPL-1.2",
       "dependencies": {
-        "@opentermsarchive/engine": "~4.0.1",
```

Comment:
Update this file as well since you have updated the package.json (for engine 5.0.3).